theano/configdefaults.py

```diff
@@ -1259,21 +1262,23 @@ def default_blas_ldflags():
             else:
                 # This branch is executed if no exception was raised
                 lib_path = os.path.join(sys.prefix, 'DLLs')
-                flags = ['-L%s' % lib_path]
+                flags = ['-L"%s"' % lib_path]
                 flags += ['-l%s' % l for l in ["mkl_core",
                                                "mkl_intel_thread",
                                                "mkl_rt"]]
                 res = try_blas_flag(flags)
                 if res:
                     return res
 
+        # to support path that includes spaces, we need to wrap it with double quotes on Windows
+        path_wrapper = "\"" if os.name == 'nt' else ""
```

Comment:
Is there a reason to do that only on nt? It would make the code easier to understand.

Also, we use sys.platform == 'win32' everywhere else. So if there isn't a reason not to use that instead of os.name, I would prefer that.

Otherwise, the PR look good. thanks.